lib/api/objectHead.js

```diff
@@ -90,6 +92,10 @@ function objectHead(authInfo, request, log, callback) {
             if (headerValResult.error) {
                 return callback(headerValResult.error, corsHeaders);
             }
+            if (locationConstraints[bucket.getLocationConstraint()]?.cold === true) {
```

Comment:
```suggestion
            if (locationConstraints[bucket.getLocationConstraint()]?.isCold) {
```
I think the value is going to be `isCold` @naren-rajendran 
Also no need for === true

Reply:
Done and updated in the test conf files: https://github.com/scality/cloudserver/pull/4504/commits/a30ae19648cf97efb17ed42ef27c84bbf91b3fff#diff-bf4e2086fcee49bbecaf2f0643e61782339817ae23eb92a6fe54de17c2654be6R95

Reply:
I also updated the same pattern in the GetObject PR: https://github.com/scality/cloudserver/pull/4506/commits/bbca89a890b79380cf128b44173164a2cc011c09